icaruscode/TPC/Calorimetry/normtools_icarus.fcl

```diff
@@ -46,25 +46,23 @@ tpcgain_sql: {
 
 tpcgain_local: {
   tool_type: NormalizeTPCLocal
-  # Recombination floating, muon+proton fit
-  # CalConstants: [81.04, 79.5, 82.88, 81.88]
-  # Recombination fixed, muon+proton fit
-  CalConstants: [80.32, 79.82, 82.24, 81.68]
+  # Relative normalization of the TPCs
+  CalConstants: [1.0118, 1.0000, 1.0333, 1.0230]
   Verbose: false
 }
 
-
 yznorm_sql: {
   tool_type: NormalizeYZSQL
   DBFileName: tpc_yz_correction_data
   DBTag: @local::ICARUS_Calibration_GlobalTags.tpc_yz_correction_data
   Verbose: false
 }
 
-# icarus_calonormtools: [@local::driftnorm, @local::yznorm, @local::tpcgain]
-icarus_calonormtools: [@local::driftnorm_sql, @local::yznorm_sql, @local::tpcgain_local]
+icarus_calonormtools: [@local::driftnorm, @local::yznorm, @local::tpcgain]
+# icarus_calonormtools: [@local::driftnorm_sql, @local::yznorm_sql, @local::tpcgain_sql]
```

Comment:
Maybe @jedori0228 or @SFBayLaser can comment on this change? I suppose it is either very right or very wrong, but I never remember which one.